### ISSUE TITLE
Remove Falco fork kernel-module patches

### ISSUE THREAD

**Issue by Stringy** [stackrox/collector#1104]
The kernel module patches are no longer needed and largely unsuitable for upstreaming. They should be removed from the falco fork to ease maintenance and reduce friction during fork updates. 

Should be completed after #1103

**Comment by Stringy** [comment]
Holding off on this until post 4.1 release for fork stability
